### ISSUE TITLE
Captcha settings no longer showing in the Craft admin.

### ISSUE THREAD

**Issue by Youniteus** [verbb/formie#266]
**Description**

The captcha settings are no longer showing in the Craft admin.

**Steps to reproduce**
1. Install plugin
2. Go to "Formie > Settings > Captcha"
3. The page show "No integrations available."

**Additional info**
- Plugin version: 1.3.4
- Craft version: 3.5.11.1
- Multi-site: No

**Additional context**
- I'm pretty sure this commit broke introduced this bug: https://github.com/verbb/formie/commit/5fa7ce6849d7610810e5b184c7d8ab731bd76f5a
- Reverting back to Formie 1.3.3 makes the captcha settings show up again.

**Comment by engram-design** [comment]
Sorry about that. Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by cookie10codes** [comment]
@engram-design when is the next release planned?

**Comment by engram-design** [comment]
@KoeketieneDesign In the next 1-2 days

**Comment by engram-design** [comment]
Fixed in [1.3.5](https://github.com/verbb/formie/releases/tag/1.3.5)
